tests/expectations/core/test_core_model_schemas.py

```diff
@@ -10,6 +12,18 @@
 expectation_dictionary = dict(core.__dict__)
 
 
+@pytest.fixture
+def safer_draft_7_validator() -> type[Draft7Validator]:
+    validator = Draft7Validator
+    validator.META_SCHEMA = {
+        **Draft7Validator.META_SCHEMA,
+        # this ensures that only specified properties are used (e.g. multipleOf, not multiple_of)
+        # otherwise, the spec says unspecified properties should be ignored
```

Comment:
Do we have docs for this schema available? Can we add a link here?

Reply:
Did you see the `README`? It's at the bottom of the list of json files.

Reply:
I added links to the test instead of this fixture